### PR TITLE
15 ACR/BLB/DSK cards

### PR DESCRIPTION
**Pull request by Fulgur14**
Added rev counters for Chainsaw and nest counters for Twitching Doll.
Added new tokens for Toby, Beastie Befriender and Twitching Doll.
Corrected a typo in Secret Plans.

Added ACR: Roshan, Hidden Magister
Added BLB: Carrot Cake, Early Winter, Might of the Meek, Pearl of Wisdon, Salvation Swan, Sunshower Druid
Added DSK: Chainsaw; Cursed Recording; Fear of Missing Out; Leyline of Hope; Screaming Nemesis; The Wandering Rescuer; Toby, Beastie Befriender; Twitching Doll

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/fear_of_missing_out.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Fear of Missing Out
+ManaCost:1 R
+Types:Enchantment Creature Nightmare
+PT:2/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDiscard | TriggerDescription$ When CARDNAME enters the battlefield, discard a card, then draw a card. If CARDNAME's spectacle cost was paid, instead discard your hand, then draw three cards.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/fear_of_missing_out.txt:5]: wrong description

Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/fear_of_missing_out.txt]: Fixed.

File: forge-gui/res/cardsfolder/upcoming/roshan_hidden_magister.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name:Roshan, Hidden Magister
+ManaCost:3 B
+Types:Legendary Creature Human Assassin
+PT:4/4
+S:Mode$ Continuous | Affected$ Creature.YouCtrl+other | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | AddType$ Assassin | Description$ Other creatures you control are Assassins in addition to their other types. The same is true for creature spells you control and creature cards you own that aren't on the battlefield.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/roshan_hidden_magister.txt:5]: bad affected

File: forge-gui/res/cardsfolder/upcoming/salvation_swan.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Name:Salvation Swan
+ManaCost:3 W
+Types:Creature Bird Cleric
+PT:3/3
+K:Flash
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self,Bird.other+YouCtrl | Execute$ TrigBlink | TriggerDescription$ Whenever CARDNAME or another Bird you control enters, exile up to one target creature you control without flying. Return it to the battlefield under its owner's control with a flying counter on it at the beginning of the next end step.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/salvation_swan.txt:7]: bad valid

File: forge-gui/res/cardsfolder/upcoming/screaming_nemesis.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Name:Screaming Nemesis
+ManaCost:2 R
+Types:Creature Spirit
+PT:3/3
+K:Haste
+T:Mode$ DamageDoneOnce | Execute$ TrigDamage | ValidTarget$ Card.Self | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME is dealt damage, it deals that much damage to any other target. If a player is dealt damage this way, they can't gain life for the rest of the game.
+SVar:TrigDamage:DB$ DealDamage | NumDmg$ X | ValidTgts$ Creature.Other,Player,Planeswalker.Other,Battle.Other | TgtPrompt$ Select any other target | RememberDamaged$ True | SubAbility DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ CantGainLife | Duration$ Permanent | RememberObjects$ Player.IsRemembered | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/screaming_nemesis.txt:8]: static missing

**Review by tool4ever — APPROVED** [review]

**Pull request merged**